recipes/glib/all/test_package/conanfile.py

```diff
@@ -18,22 +17,31 @@ def requirements(self):
         self.requires(self.tested_reference_str)
 
     def build_requirements(self):
-        if self.settings.os != "Windows":
+        if self.settings.os != "Windows" and not self.conf.get("tools.gnu:pkg_config", default=False, check_type=str):
             self.tool_requires("pkgconf/1.9.3")
 
     def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["CMAKE_CROSSCOMPILING"] = cross_building(self)
```

Comment:
shouldn't be needed

Reply:
The [documentation](https://cmake.org/cmake/help/latest/variable/CMAKE_CROSSCOMPILING.html) on `CMAKE_CROSSCOMPILING` lists some shortcomings which is why I set it explicitly here.

Reply:
it's useless because it's not used anyway in CMakeLists

Reply:
At some point I was... maybe that was lost during the rebase or I just forgot. I'll remove it.